docs/core/tools/global-json.md

```diff
@@ -1,41 +1,64 @@
 ---
 title: global.json reference
-description: See the schema for the global.json file, which permits setting the .NET Core tools version.
-keywords: .NET, .NET Core
-author: blackdwarf
+description: See the schema for the global.json file, which permits setting the .NET Core SDK version.
+author: mairaw
 ms.author: mairaw
-ms.date: 04/05/2017
+ms.date: 03/19/2018
 ms.topic: article
 ms.prod: .net-core
 ms.technology: dotnet-cli
-ms.devlang: dotnet
-ms.assetid: 96102f96-d403-4385-8ef6-5d80e406eb0c
 ms.workload: 
   - dotnetcore
 ---
-
 # global.json reference
 
-The *global.json* file allows selection of the .NET Core tools version being used through the `sdk` property.
+[!INCLUDE [topic-appliesto-net-core-all](../../../includes/topic-appliesto-net-core-all.md)]
+
+The *global.json* file allows selection of the .NET Core SDK version being used through the `sdk` property.
 
-.NET Core CLI tools look for this file in the current working directory (which isn't necessarily the same as the project directory) or one of its parent directories.
+.NET Core SDK looks for this file in the current working directory (which isn't necessarily the same as the project directory) or one of its parent directories.
 
 ## sdk
+
 Type: Object
 
-Specifies information about the SDK.
+Specifies information about the .NET Core SDK.
 
 ### version
+
 Type: String
 
-The version of the SDK to use.
+The version of the .NET Core SDK to use.
 
-For example:
+Note that this field:
+
+- Doesn't have globbing support, that is, the full version number has to be specified.
+- Doesn't support version ranges.
+
+The following example shows the contents of a *global.json* file:
 
 ```json
 {
   "sdk": {
-    "version": "1.0.0-preview2-003121"
+    "version": "2.1.300-preview1-008174"
   }
 }
 ```
+
+You can easily create this file by executing the [dotnet new](dotnet-new) command as follows:
+
+```console
+dotnet new globaljson --sdk-version 2.1.300-preview1-008174
+```
+
+## Matching rules
```

Comment:
@mairaw 
The description is nearly correct; any use of the term: "roll-forward" should not be used.

## Matching rules

With .NET Core SDK 1.x, if you specified a version and no exact match was found, the latest installed SDK version was used. Latest SDK version can be either release or pre-release - the highest version number wins.

Starting with .NET Core SDK 2.0, the following rules apply when determining which version of the SDK to use:

- If *global.json* doesn't contain a version or a *global.json* file cannot be found, the latest installed SDK version is used. Latest SDK version can be either release or pre-release - the highest version number wins.
- If *global.json* does contain a specified SDK version:
  - If the specified SDK version is found on the machine, that exact version is used.
  - If the specified SDK version cannot be found on the machine, the latest installed SDK **patch-version** of the specified SDK version is used. Latest installed SDK **patch-version** can be either release or pre-release - the highest version number wins. For more information about versioning, see [Semantic versioning](../versions/index.md/#semantic-versioning).
  - If the specified SDK version and an appropriate SDK **patch-version** cannot be found, an error is thrown.

Reply:
Let's remove the sentence with semantic versioning since we are not using it. 

Also, the language around the global.json file sort of makes it sound like it should be there. Let's change this

If global.json doesn't contain a version or a global.json file cannot be found

to 

If no global.json file is found, or the global.json doesn't contain a version

Reply:
If no global.json file is found, or the global.json doesn't contain a specified SDK version

Reply:
@mairaw 
This should be ready. I've edited the "Matching rules" comment above - it should just cut 'n paste.

Reply:
Thank you @johnbeisner. I'll look at the feedback today and get this PR updated. 

There was also a missing piece of this PR that is related to @nguerrera comment here: https://github.com/dotnet/docs/issues/2902#issuecomment-322318967

It wasn't super clear to me how the VS/MSBuild case works and what happens when with the multiple possible combinations (VS versions earlier than 15.3 and later, SDK 1.1, SDK 2.0, etc.)

Reply:
I changed the wording a bit and tried to add some explanation of what the version number looks like and what constitutes a patch version.

Reply:
As someone new to .Net core it isn't clear to me what a patch version is? Is that the third number of set. I.e in V2.1.200 the patch would be 200? If I specify 2.1 would I get 2.1.200 or 2.1.300-rc1? Maybe some examples would help.

Reply:
I'm adding that info @martingbrown. For 2.1.300+, it's the last 2 digits of the last set.